tests/viz/_parity_runner.js

```
@@ -0,0 +1,28 @@
+
+const ROOT = process.argv[2];
+if (!ROOT) {
+    console.error('usage: node _parity_runner.js <repo-root>');
+    process.exit(1);
+}
+
+const ASSETS = path.join(ROOT, 'src', 'hypergraph', 'viz', 'assets');
+
+// derivation.js attaches HypergraphDerivation; scene_builder.js attaches
+// HypergraphSceneBuilder. Evaluate both in the current global scope.
+eval(fs.readFileSync(path.join(ASSETS, 'derivation.js'), 'utf-8'));
```

Biome (2.4.13)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)